README.md

```diff
@@ -1,32 +1,46 @@
 # LCache
 Foundation Library for Coherent, Multi-Layer Caching
 
-## Testing
-
 [![Build Status](https://travis-ci.org/lcache/lcache.svg?branch=master)](https://travis-ci.org/lcache/lcache)
 [![Coverage Status](https://coveralls.io/repos/github/lcache/lcache/badge.svg?branch=master)](https://coveralls.io/github/lcache/lcache?branch=master)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/lcache/lcache/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/lcache/lcache/?branch=master)
+[![Documentation](https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat)](https://lcache.github.io/lcache/)
+[![API](https://img.shields.io/badge/api-latest-brightgreen.svg?style=flat)](https://lcache.github.io/lcache/api/master)
+[![License](https://poser.pugx.org/lcache/lcache/license)](https://packagist.org/packages/lcache/lcache)
 
-### On Fedora
+## Testing
 
  1. Install packages:
+ 
+    ** On Fedora: **
 
     ```
     sudo dnf install -y php-cli composer php-phpunit-PHPUnit php-phpunit-DbUnit php-pecl-apcu
```

Comment:
We should add `php-pecl-xdebug` and `php-opcache` for parity with the Brew list.